packages/nextjs/app/debug/_components/contract/Struct.tsx

```diff
@@ -104,15 +104,27 @@ export const Struct = ({
                   index,
                 );
                 return (
-                  <ContractInput
-                    setFormErrorMessage={setFormErrorMessage}
-                    abi={abi}
-                    setForm={setForm}
-                    form={form}
-                    key={index}
-                    stateObjectKey={key}
-                    paramType={variant}
-                  />
+                  <div key={index} className="flex items-center gap-3">
+                    <input
+                      type="checkbox"
+                      name={`radio-${index}`}
+                      className="radio radio-xs radio-secondary"
+                      checked={index === activeVariantIndex}
+                      onClick={() => {
```

Comment:
for an checkbox input the action callback should be `onChange`? or keep onClick and add an empty onChange.

we can see the error msg:
```
You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`. 
```

Reply:
i will go for the latter for a quicker fix